PROJECT

```diff
@@ -12,6 +12,9 @@ resources:
   kind: ExternalSecret
   version: v1alpha1
 - group: external-secrets
+  kind: ClusterExternalSecret
+  version: v1alpha1
+version: "2"
```

Comment:
i think this line doesn't belong here :thinking:

Reply:
Removed these lines!